force-di/main/classes/di_PlatformCache.cls

```diff
@@ -107,91 +247,62 @@ public with sharing class di_PlatformCache
         pushCacheKeyIndexMapToCache();
     }
 
-    private static Map<String, String> generatedKeyNames = new Map<String, String>();
-
+    @TestVisible
     private String constructKeyName( Schema.SObjectType bindingSObjectType, String developerName )
     {
-        String key = ( ( bindingSObjectType != null ) ? bindingSObjectType.getDescribe().getName().toLowerCase().replaceAll('__','') : '' ) 
-                            + ( String.isBlank(developerName) ? '' : developerName.toLowerCase().trim() );
+        String key = ( ( bindingSObjectType != null ) ? bindingSObjectType.getDescribe().getName().toLowerCase().replaceAll('__','') : '' )
+                     + ( String.isBlank(developerName) ? '' : developerName.toLowerCase().trim() );
 
         // put generated hash into a map on first pass, so that we do not perform hashcode() operation more than once per "key"
         // hashcode() is a more expensive operation than map.get()
         if (generatedKeyNames.containsKey(key)) { return generatedKeyNames.get(key); }
 
         String hash = String.valueOf( Math.abs( ( key ).hashcode() ) );
-        //System.debug('Creating Hash For => ' + developerName + ' && ' + bindingSObjectType + ' := ' + hash);
+        //log('Creating Hash For => ' + developerName + ' && ' + bindingSObjectType + ' := ' + hash);
```

Comment:
I'm not sure what the intention is with this commented log statement.  If needed then uncomment, otherwise should it be deleted?

Reply:
I am not a big fan of system.debug left in code. However, this is not my code so I was on the fence. The goal was .. if you must, then localize within and refactor later. There is some other code commented out that had used 'system.debug' and I replaced with log ... Still not ideal but I felt it cleaned it up a small bit. Again, system.debug should at least be centralized. Yes, I agree with the S in SOLID but the initial intent was code coverage.

Reply:
Good day @bjanderson70 !
I am really excited about seeing the Code Coverage part of this pull request to be merged.
The above comment is good the learn!
Is it okay to remove the "System.debug" changes of di_PlatformCache.cls above and create a separate Pull Request for this (probably all System.debug related items in one Pull Request)?